website/docs/cli/workspaces/index.mdx

```diff
@@ -7,72 +7,99 @@ description: >-
-collection of resources you manage.
+Workspaces in the Terraform CLI refer to separate instances of [state data](/language/state) inside the same working directory.
+
+Terraform relies on state to associate resources with real-world objects. When you run the same configuration multiple times with separate state data, Terraform can manage multiple sets of non-overlapping resources.
+
+Workspaces let you create many instances of the same infrastructure or inject different [variable values](/language/values/variables) for each instance. For example, you may want to quickly create infrastructure to test changes without affecting the production deployment.
+
+## Managing CLI Workspaces
+
+Every [initialized working directory](/cli/init) starts with one workspace named `default`.
+
+Use the [`terraform workspace list`](/cli/commands/workspace/list), [`terraform workspace new`](/cli/commands/workspace/new), and [`terraform workspace delete`](/cli/commands/workspace/delete) commands to manage the available workspaces in the current working directory.
+
+Use [the `terraform workspace select` command](/cli/commands/workspace/select) to change the currently selected workspace. For a given working directory, you can only select one workspace can be at a time. Most Terraform commands only interact with the currently selected workspace. This includes [provisioning](/cli/run) and [state manipulation](/cli/state).
+
+
+## Use Cases
+
+You can create multiple [working directories](/cli/init) to maintain multiple instances of a configuration with completely separate state data. However, Terraform installs a separate cache of plugins and modules for each working directory, so maintaining multiple directories can waste bandwidth and disk space. This approach also requires extra tasks like updating configuration from version control for each directory separately and reinitializing each directory when you change the configuration. Workspaces are more convenient because they let you create different sets of infrastructure with the same working copy of your configuration and the same plugin and module caches.
+
+A common use for multiple workspaces is to create a parallel, distinct copy of
+a set of infrastructure to test a set of changes before modifying production infrastructure.
+
+Non-default workspaces are often related to feature branches in version control.
+The default workspace might correspond to the `main` or `trunk` branch,
+which describes the intended state of production infrastructure. When a
+feature branch is created to develop a change, the developer of that feature
+might create a corresponding workspace and deploy into it a temporary copy
+of the main infrastructure so that they can test changes without affecting
+the production infrastructure. Once the change is merged and deployed to the
+default workspace, they destroy the test infrastructure and delete the temporary workspace.
```

Comment:
These paragraphs are still hard-wrapped like the old content on this page. Did you intend to remove all of the hard-wrapping from the paragraphs here? (I've noticed you've been standardizing on not hard-wrapping in other pages you've been updating, so I'm guessing this was just an oversight...)

Reply:
oh yuck yes - thank you I'm going to fix this :)